### PR TITLE
Added release notes for 4.1.9

### PR DESCRIPTION
**Pull request by danielvaughan**

### CODE REVIEW

File: webapps/core/src/main/asciidoc/releasenotes.adoc [hunk 1]
```diff
@@ -3,6 +3,20 @@
 
 This pages contains links to release notes for BioSamples for version 4.0.0 and higher. This release represents a comprehensive overhaul and therefore previous release notes are no longer applicable.
 
+[[section]]
+4.1.9
+----
+* Added a Curation Undo Pipeline to allow for removal of erroneous curations.
+* Fix an issue where long attributes break the sample box UI. 
+
+
```
Comment (mcourtot) [inline webapps/core/src/main/asciidoc/releasenotes.adoc]: looks good to me

File: pipelines/curationundo/src/main/java/uk/ac/ebi/biosamples/curationundo/CurationUndoCallable.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        this.canary = canary;
+        LocalDate localDate = LocalDate.parse("2018-10-23");
+        LocalDateTime localDateTime = localDate.atStartOfDay();
+        instant = localDateTime.toInstant(ZoneOffset.UTC);
+    }
+
+    public static final String[] NON_APPLICABLE_SYNONYMS = {"n/a", "na", "n.a", "none",
+            "unknown", "--", ".", "null", "missing", "[not reported]",
+            "[not requested]", "not applicable", "not_applicable", "not collected", "not specified", "not known", "not reported"};
+
```
Comment (mcourtot) [inline pipelines/curationundo/src/main/java/uk/ac/ebi/biosamples/curationundo/CurationUndoCallable.java]: should those be defined globally somewhere else? We're using them in the curation pipeline as well, so as is would require update in 2 places when we want to make changes?

Comment (danielvaughan) [reply pipelines/curationundo/src/main/java/uk/ac/ebi/biosamples/curationundo/CurationUndoCallable.java]: It is only meant to be a one-use pipeline.

Comment (danielvaughan) [reply pipelines/curationundo/src/main/java/uk/ac/ebi/biosamples/curationundo/CurationUndoCallable.java]: I did not want to make any changes to existing code or create any dependencies between packages

**Comment by danielvaughan** [comment]
You are right @theisuru, I just ran the commands in the instructions instead of reading them :)

**Pull request merged**